app/views/tasks/edit.html.erb

```diff
@@ -0,0 +1,9 @@
+<p id="notice" style="color: #730E15"><%= notice %></p>
+
+<%= link_to "Project", @task.project %>
+
+<h1>Editing Task</h1>
+
+<%= render 'form', task: @task, project: @task.project %>
+
+<%= link_to 'Back', projects_path %>
```

Comment:
Incorrect link. We should back to project tasks

Reply:
Oops, i missed this somehow. Fixed it